### PR TITLE
Remove deprecated heating parameters from input files

### PR DESCRIPTION
**Pull request by gassmoeller**
This removes the deprecated heating parameters (since Hackathon 2015) from all our input files. The options itself for now stay in the code. I am in favor of removing them after the next release. The changes were done by a script, and I need to test first if any tests break. Also some input files will now have comments that reference parameters that are not longer in the input files, and I need to take a look at all of those .part. ... files. Not ready for review yet.

### CODE REVIEW

**Comment by gassmoeller** [comment]
I updated the cookbooks and removed comments that referenced the now gone/moved parameters. Ready for review.

File: doc/manual/manual.tex [hunk 1]
```diff
@@ -524,8 +524,10 @@ \subsubsection{A comment on adiabatic heating}
   & \approx -\alpha \rho T \mathbf u \cdot \mathbf g.
 \end{align*}
 While this simplification is possible, it is not necessary if you have access
-to the total pressure. \aspect{} therefore implements the original term
-without this simplification.
+to the total pressure. \aspect{} therefore by default implements the original 
+term without this simplification, but allows to simplify this term by setting
+the \textit{Use simplified adiabatic heating}
```
Comment (bangerth) [inline doc/manual/manual.tex]: We seem to consistently use `\texttt` for parameter names. 

Would it also be worthwhile adding a link to the corresponding section of this parameter?

**Comment by tjhei** [comment]
Looks good to me.

**Comment by gassmoeller** [comment]
I made the change. Could someone take a look if the manual compiles without error? I currently have no access to the special packages we use, and the syntax is a bit obscure. Otherwise ready from my side.

**Comment by tjhei** [comment]
> Could someone take a look if the manual compiles without error?

Looks good.

**Pull request merged**